core/src/main/scala/org/scalarelational/instruction/Query.scala

```diff
@@ -55,4 +59,43 @@ case class Query[Expressions, Result](expressions: Expressions,
   def async = table.datastore.async {
     result
   }
+
+  def to[R](implicit manifest: Manifest[R]) = {
```

Comment:
No, it would still be necessary because ``InsertSingle`` and ``InsertMultiple`` don't behave in the same way and need special treatment in ``SQLDatastore``.